docs/area-owners.md

```diff
@@ -81,7 +81,7 @@ Note: Editing this file doesn't update the mapping used by the `@msftbot` issue
 | area-System.Diagnostics.EventLog               | @jeffhandley  | @adamsitnik @carlossanlop @jozkee                   |                                                                                                                                                                                                                                                                                           |
 | area-System.Diagnostics.PerformanceCounter     | @jeffhandley  | @adamsitnik @carlossanlop @jozkee                   |                                                                                                                                                                                                                                                                                           |
 | area-System.Diagnostics.Process                | @jeffhandley  | @adamsitnik @carlossanlop @jozkee                   |                                                                                                                                                                                                                                                                                           |
-| area-System.Diagnostics.Tracing                | @tommcdon     | @noahfalk @tommcdon @Anipik @ViktorHofer @tarekgh   | Packages: <ul><li>System.Diagnostics.DiagnosticSource</li><li>System.Diagnostics.PerformanceCounter - @Anipik @ViktorHofer</li><li>System.Diagnostics.Tracing - @Anipik @ViktorHofer</li><li>System.Diagnostics.TraceSource</li></ul>                                                     |
```

Comment:
System.Diagnostics.PerformanceCounter has its own area now, so I assume that it should not be mentioned here.